libvirt/tests/src/libvirt_vcpu_plug_unplug.py

```
@@ -298,6 +304,28 @@ def check_setvcpus_result(cmd_result, expect_error):
             vmxml.set_agent_channel()
         else:
             vmxml.remove_agent_channels()
+        if cpu_mode:
+            logging.info("Setting CPU mode to: %s", cpu_mode)
+            try:
+                cpu_xml = vmxml.cpu
+            except xcepts.LibvirtXMLNotFoundError:
```

Ruff (0.15.17)

[error] Undefined name `xcepts`

(F821)